### PR TITLE
admonition french translation

### PR DESCRIPTION
**Pull request by bull0n**
Translation for french for the admonition file.

Let me know if something isn't okay :)

Issue #230

### CODE REVIEW

File: templates/fr/admonition.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+À qui de droit :
+
+Le {request_date} je vous ai envoyé une requête en accord avec l'article {request_article} GDPR.
+{Description du problème, par exemple: Je n'ai malheureusement pas reçu de réponse. La période d'un mois selon l'article 12(3) alinéa 1 GDPR est donc dépassée.}
+
+C'est pourquoi je vous contacte une nouvelle fois. Après l'arrivée de cette admonition, veuillez répondre dans un délai de 2 semaines. Autrement je garde le droit de prendre des actions légales contre vous et déposer plainte avec les authorités responsables. 
+on {request_date} I sent you a request according to Article {request_article} GDPR.
```
Comment (baltpeter) [inline templates/fr/admonition.txt]: You forgot to delete this line. :)

File: templates/fr/admonition.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Très sincèrement,
```
Comment (baltpeter) [inline templates/fr/admonition.txt]: In the French access template by @deschantkn, we are using "Cordialement" instead.

My French is unfortunately a bit rusty, so I don't know which one is more appropriate but I would like to be consistent between the templates. What do you think?

File: templates/fr/admonition.txt (hunk 1 above)
Comment (bull0n) [reply templates/fr/admonition.txt]: Oops sorry >.<, gonna correct that

File: templates/fr/admonition.txt (hunk 2 above)
Comment (bull0n) [reply templates/fr/admonition.txt]: Oh yes, I should have checked the other files.

**Comment by bull0n** [comment]
I changed the 2 mistakes !

**Comment by baltpeter** [comment]
Awesome, thanks again!

Not sure if you've seen this already but feel free to signup to our Hacktoberfest event to get some stickers from us, if you want: https://www.datarequests.org/blog/hacktoberfest-2019/

**Pull request merged**